eZ/Publish/Core/Repository/Tests/Service/Mock/PermissionTest.php

```diff
@@ -1075,34 +1074,16 @@ protected function getUserReferenceMock()
     /**
      * @return \eZ\Publish\API\Repository\Repository|\PHPUnit\Framework\MockObject\MockObject
      */
-    protected function getRepositoryMock($methods = [])
+    protected function getRepositoryMock(): Repository
```

Comment:
This was quite misleading as it was not used. Moreover inherited from parent `Base` which did not have this argument.